.github/workflows/deploy_with_labels.yml

```
@@ -0,0 +1,97 @@
+        id: latest_commit
+        run: |
+          LATEST_COMMIT=$(git rev-parse origin/auth_rbac)
+          echo "LATEST_COMMIT=$LATEST_COMMIT" >> $GITHUB_ENV
+
+      - name: Compare commits and trigger build if different
+        if: env.LATEST_COMMIT != env.LAST_SUCCESSFUL_COMMIT
+        run: echo "New commits found. Proceeding with build and deployment."
+
+      - name: Set up Docker Buildx
+        if: env.LATEST_COMMIT != env.LAST_SUCCESSFUL_COMMIT
+        uses: docker/setup-buildx-action@v2
```

actionlint (1.7.4)

the runner of "docker/setup-buildx-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)